src/explore-education-statistics-admin/src/services/releaseService.ts

```diff
@@ -199,9 +204,20 @@ const releaseService = {
     return client.post(`/releases/${releaseId}/status`, createRequest);
   },
 
+  async getDeleteReleasePlan(releaseId: string): Promise<DeleteReleasePlan> {
+    const deletePlan = await client.get<Omit<DeleteReleasePlan, 'releaseId'>>(
+      `/release/${releaseId}/delete-plan`,
+    );
+    return {
+      ...deletePlan,
+      releaseId,
```

Comment:
If we pass the `releaseId` to this function, do we need to also include it in the response? Seems redundant as the caller should know the argument it used to call the function with

Reply:
Shifted the holding on of the releaseId to the caller rather than keeping a hold of it here.